module-system/module-implementations/sov-bank/src/token.rs

```diff
@@ -164,14 +164,14 @@ impl<C: sov_modules_api::Context> Token<C> {
         Ok(())
     }
 
-    /// Mints a given `amount` of token sent by `sender` to the specified `minter_address`.
+    /// Mints a given `amount` of token sent by `sender` to the specified `mint_to_address`.
     /// Checks that the `authorized_minters` set is not empty for the token and that the `sender`
-    /// is an `authorized_minter`. If so, update the balances of token for the `minter_address` by
+    /// is an `authorized_minter`. If so, update the balances of token for the `mint_to_address` by
     /// adding the minted tokens. Updates the `total_supply` of that token.
-    pub(crate) fn mint(
+    pub(crate) fn mint_from_eoa(
```

Comment:
To follow similar naming to bank, shouldn't this be `mint`, and `sender` argument should be `authorizer`? Given that we're trying to enable "sender" to not always be the EOA that initiated the transaction?

Reply:
Oops, yeah I did an automatic refactor here and didn't review. Fixing!